packages/frontend/src/components/connect-button/account-menu/index.tsx

```diff
@@ -46,6 +48,14 @@ export function AccountMenu({
     const rootRef = useRef(null);
     const { disconnect } = useDisconnect();
 
+    const [copied, setCopied] = useState(false);
+
+    const handleCopyClick = () => {
+        navigator.clipboard.writeText(account.address).then(() => {
```

Comment:
From `caniuse` I see that this has roughly 93% of users support. I'm wondering if we should use a library to guarantee better compatibility or at least make this a bit more sophisticated.